### ISSUE TITLE
Test FCNC variables and hyperparameters

### ISSUE THREAD

**Issue by asalvadorsalas** [asalvadorsalas/IFAE_NNTools#6]
Test NN with variables and hyp form [Hplus->cb](https://cds.cern.ch/record/2683620/files/ATL-COM-PHYS-2019-923.pdf?)

- [x] First test with variables

reframelep_phi
reframelep_eta
mbb_leading_bjets
mbb_maxdr
mbb_mindr
m_jj_leading_jets

- [x] Later with the hyperparameters
Hidden layers-> 4
Nodes each -> 196
dropout -> 40%
batch size -> 2048
learning rate -> 0.00428095

- [x] Test with leadingjet pT 
Test the inclusion of a global variable which I think is lacking in the NN
